letsdrill.rb

```diff
@@ -1,13 +1,34 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  if integer >= 90 then
```

Comment:
Awesome work! There are some styling things going on here so I wanted to link the style guide for reference: https://github.com/rubocop-hq/ruby-style-guide

Using `'` single tick quotes when you aren't interpolating.
Don't need to explicitlty call `return` to return values, ruby will evaluate the last line of the method as is and return it.

Functionality wise here this work but does it seem like it's starting to get a little lengthy? If you have a hunch you are correct, check out case statements and also you can have the final catch in your `if/elsif/else` be an `else` instead of the return outside of the `if` statement